test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs

```diff
@@ -24,6 +24,7 @@ namespace NuGet.XPlat.FuncTest
     public class XPlatAddPkgTests
     {
         private static readonly string projectName = "test_project_addpkg";
+        private static MSBuildAPIUtility msBuild = new MSBuildAPIUtility(new TestCommandOutputLogger());
```

Comment:
I would avoid sharing this between tests, you could make this a property that returns a new instance each time if needed though.